core/app/assets/javascripts/spree.js

```diff
@@ -22,15 +22,23 @@ Spree.pathFor = function (path) {
 }
 
 Spree.localizedPathFor = function(path) {
-  if (typeof (SPREE_LOCALE) !== 'undefined') {
+  if (typeof (SPREE_LOCALE) !== 'undefined' && SPREE_CURRENCY !== 'undefined') {
```

Comment:
We have to check `typeof` variable, instead of variable itself
```suggestion
  if (typeof (SPREE_LOCALE) !== 'undefined' && typeof SPREE_CURRENCY !== 'undefined') {
```